exasol_integration_test_docker_environment/doctor.py

```diff
@@ -0,0 +1,66 @@
+"""
+The doctor module provides functionality to check the health of the `exasol_integration_test_docker_environment`
+package and also provide help to find potential fixes.
+"""
+from enum import Enum
+from typing import Iterator
+
+import docker
+from docker.errors import DockerException
+
+
+class ErrorCodes(Enum):
```

Comment:
We need to replace this at some point with https://github.com/exasol/error-reporting-python. Please create ticket, that we introduce error-reporting-python.

Reply:
:heavy_check_mark: https://github.com/exasol/integration-test-docker-environment/issues/184